### PR TITLE
Feature requirements

### PR DESCRIPTION
**Pull request by medikornov**

### CODE REVIEW

File: Pages/FoodCustomer/Cart.cshtml.cs [hunk 1]
```diff
@@ -111,30 +109,53 @@ public IActionResult OnPostReserve()
...
-                    foodPrice = food.Food.CurrentPrice.DiscountPrice;
-                }
+                    foodId = food.ID,
+                    foodName = food.FoodName,
+                    quantity = crt.Quantity,
+                    foodsOwner = food.ApplicationUserId
+                }).ToList();
+
+            // group join
+            var infoForCustomer = _context.ApplicationUsers.ToList().GroupJoin(
```
Comment (0rl1s) [inline Pages/FoodCustomer/Cart.cshtml.cs:126]: Used groupJoin. Requirement +. Well done.

File: Pages/Login.cshtml.cs [hunk 2]
```diff
@@ -26,6 +27,7 @@ public void OnGet()
         {
         }
 
+        // optional argument usage
```
Comment (0rl1s) [inline Pages/Login.cshtml.cs:30]: And one more requirement +

File: Pages/FoodCustomer/Cart.cshtml.cs [hunk 3]
```diff
@@ -21,7 +21,7 @@ namespace FoodKept.Pages.FoodCustomer
     [Authorize(Roles = "Customer, Admin")]
     public class CartModel : PageModel
     {
-        public IList<ShoppingCart> Cart { get; set; }
```
Comment (ArmantasPik) [inline Pages/FoodCustomer/Cart.cshtml.cs:24]: I think that our naming is to name properties capitalised. We'll discuss.

**Review by ArmantasPik — APPROVED** [review]
Everything works, but some things need to be looked into.

**Pull request merged**